src/providers/DevicesProvider/AudioInputProvider.tsx

```diff
@@ -67,16 +71,20 @@ const AudioInputProvider: React.FC = ({ children }) => {
           console.log(
             'Previously selected audio input lost. Selecting a default device.'
           );
-          meetingManager.selectAudioInputDevice(newAudioInputs[0].deviceId);
+          await meetingManager.selectAudioInputDevice(newAudioInputs[0].deviceId);
 
           // Safari and Firefox don't have this "default" as device Id
           // Only Chrome have this "default" device
         } else if (selectedInputRef.current === 'default') {
           console.log(
             `Audio devices updated and "default" device is selected. Reselecting input.`
           );
+          let currentDevice: Device | AudioTransformDevice = 'default';
+          if (reselection) {
+            currentDevice = await reselection(currentDevice);
+          }
           try {
-            await audioVideo?.chooseAudioInputDevice(selectedInputRef.current);
+            await meetingManager.selectAudioInputDevice(currentDevice);
```

Comment:
Was this a bug?

Reply:
I think the previous code works without VF logic. But after we integrate the VF feature, we need to call meeting manager instead of `audioVideo`